Reader.py

```diff
@@ -1,13 +1,27 @@
 from CharacterData import CharacterData
 
-def read_infile(infile):
+def read_infile(filepath):
+	"""
+		DEPRECATED : TO BE MOVED INTO GAMEDATA CLASS
+	"""
 	state = 0
 	game_characters = [] # list of characters and their game data
 	max_stats = {} # nested dictionary {class: {stat name: stat}}
 	promotion_gains = {} # nested dictionaries {base_class: {promoted_class: {stat name: {stat gains/losses}}}}
 	base_classes = [] # list of base classes
 	promoted_classes = [] # list of promoted classes
 	
+	try:
+		infile = open(filepath, "r")
+	except:
```

Comment:
It is not good practice to have a general except statement. Please fill in the specific case you will be excepting